draft-ietf-intarea-provisioning-domains.md

```diff
@@ -778,6 +780,8 @@ It is expected that for some years, networks will have a mixed
 environment of PvD-aware hosts and non-PvD-aware hosts. If there is a
 need to give specific information to PvD-aware hosts only, then it is
 RECOMMENDED to send two RA messages, one for each class of hosts.
+This approach allows for two distinct sets of configuration information
+to be sent in a way that will not disrupt non-PvD-aware hosts.
```

Comment:
...and lowers the risk of approaching an MTU limit (from, say, duplicated information)

Reply:
Good call, added a sentence!